core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -142,7 +147,10 @@ class DynamicConfigManager(private val zkUtils: ZkUtils,
       val fullSanitizedEntityName = entityPath.substring(index + 1)
 
       val entityConfig = AdminUtils.fetchEntityConfig(zkUtils, rootEntityType, fullSanitizedEntityName)
-      logger.info(s"Processing override for entityPath: $entityPath with config: $entityConfig")
+      val loggableConfig = entityConfig.asScala.map {
```

Comment:
@ijuma Yes, version 1 doesn't support SCRAM or any paths involving `users`. I had tested using command line, but looking at the SCRAM integration test, it could also do dynamic update for client credentials so that this path is exercised. I have updated the test.